frontend/catalyst/__init__.py

```diff
@@ -39,7 +39,6 @@
 
 try:
     if INSTALLED:
-        # pylint: disable=no-name-in-module
```

Comment:
I think this needs to stay because in some configurations `_revision` doesn't exist?

Reply:
Unfortunately it is not possible to have no warnings at all because of the substantial differences in the environments (like having build artifacts or not, being installed from wheel or being in a development environment, etc), but there are different severities in the issues pylint raises.
For instance the _warning_ about unnecessary exclusions can be safely ignore because it will not produce an error code (i.e. CI will not fail on this), while not having the exclusion and triggering a `no-name-in-module` _error_ would produce an error code.